From: Anthony Tseng <darkdh@gmail.com>
Date: Wed, 1 Aug 2018 11:34:35 -0700
Subject: Add UnRegister to WebsiteSettingsRegistry

Original subject: Use BraveInit() to unregister and register the default registry value

~~~
CalyxOS porting note: This change comes originally from brave-core:
https://github.com/brave/brave-core/commit/8cdd6dc31fd198ab2db38b1579aba1819fa3c303

Original authorship has been retained.
- tommy
~~~

Change-Id: If2f3dd60cd4d96a0022702582b6f57f49189dac3
---
 .../content_settings/core/browser/website_settings_registry.h    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/content_settings/core/browser/website_settings_registry.h b/components/content_settings/core/browser/website_settings_registry.h
--- a/components/content_settings/core/browser/website_settings_registry.h
+++ b/components/content_settings/core/browser/website_settings_registry.h
@@ -78,6 +78,7 @@ class WebsiteSettingsRegistry {
       WebsiteSettingsInfo::ScopingType scoping_type,
       Platforms platforms,
       WebsiteSettingsInfo::IncognitoBehavior incognito_behavior);
+  void UnRegister(ContentSettingsType type) {website_settings_info_.erase(type);}
 
   const_iterator begin() const;
   const_iterator end() const;
-- 

